pkg/api/api.go

```diff
@@ -87,6 +89,10 @@ func newApi(id string, apiPath string) Api {
 }
 
 func NewApi(id string, apiPath string) Api {
+
+	// TODO log warning if the user tries to create an API with a id not present in map above
```

Comment:
That is currently only possible if changing the code though, isn't it?

Reply:
I'm afraid, no. Any user can import a package (pkg/` folder) in an external application and use it. I guess we should really start with validating the arguments passed to our public API. That's not just here the case, but everywhere across `monaco`.

Reply:
ah, good point :+1: